src/containers/LtiModel/index.js

```diff
@@ -103,6 +104,7 @@ const LTIProjectShared = (props) => {
                                     setActivePlaylist(counterPlaylist + counterTop + 1);
                                   }
                                   setActiveActivity(null);
+                                  setDefaultPlaylist(data2.id);
                                   setFormData({
                                     name: data2.title,
                                     _id: data2.id,
```

Comment:
Are we using variables _id now? Or we are following some name conventions for variables / keys ?

Reply:
updated